### PR TITLE
Adding a stable 2019-12-01 API version for licenses.

### PR DESCRIPTION
**Pull request by sornaks**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Review by KrisBash — APPROVED** [review]
Unchanged from prior, and no known issues called out.  Signing off for ARM

**Comment by ArcturusZhang** [comment]
Hi @sornaks if you want go SDK, please add a new entry of the new tag in `readme.go.md` file.

File: specification/softwareplan/resource-manager/readme.go.md [hunk 1]
```diff
@@ -17,3 +17,12 @@ Please also specify `--go-sdks-folder=<path to the root directory of your azure-
...
 output-folder: $(go-sdk-folder)/services/preview/$(namespace)/mgmt/2019-06-01-preview/$(namespace)
 ```
+
+### Tag: package-2019-12-01 and go
+
+These settings apply only when `--tag=package-2019-12-01 --go` is specified on the command line.
+Please also specify `--go-sdks-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+```yaml $(tag) == 'package-2019-12-01' && $(go)
+output-folder: $(go-sdk-folder)/services/preview/$(namespace)/mgmt/2019-12-01/$(namespace)
```
Comment (ArcturusZhang) [inline specification/softwareplan/resource-manager/readme.go.md]: Please add a `preview` suffix to the api-version, ie, change the output folder to:
`$(go-sdk-folder)/services/preview/$(namespace)/mgmt/2019-12-01-preview/$(namespace)`

**Review by ArcturusZhang — CHANGES_REQUESTED** [review]

File: specification/softwareplan/resource-manager/readme.go.md (hunk 1 above)
Comment (ArcturusZhang) [reply specification/softwareplan/resource-manager/readme.go.md]: Sorry, to confirm, is this version a stable version? If this is stable, the output-folder should be:
`$(go-sdk-folder)/services/$(namespace)/mgmt/2019-12-01/$(namespace)`
If this is preview, please ignore this comment

Comment (sornaks) [reply specification/softwareplan/resource-manager/readme.go.md]: This is a stable version. let me update the path

**Review by ArcturusZhang — APPROVED** [review]

**Comment by ArcturusZhang** [comment]
Hi @qiaozha could you please help on the js generation failure?

**Comment by qiaozha** [comment]
@sornaks please follow the instructions as I've told you in Teams.

**Pull request merged**